conference.js

```diff
@@ -490,13 +495,31 @@ export default {
      * on.
      * @param {boolean} options.startScreenSharing=false - if <tt>true</tt>
      * should start with screensharing instead of camera video.
+     * @param {boolean} options.startWithAudioMuted - will start the conference
+     * without any audio tracks.
+     * @param {boolean} options.startWithVideoMuted - will start the conference
+     * without any video tracks.
      * @returns {Promise.<JitsiLocalTrack[], JitsiConnection>}
      */
     createInitialLocalTracksAndConnect(roomName, options = {}) {
         let audioAndVideoError,
             audioOnlyError,
             screenSharingError,
             videoOnlyError;
+        const initialDevices = [];
+        let requestedAudio = false;
+        let requestedVideo = false;
```

Comment:
I think it will help if we add a line of comments for those explaining how we're going to use them.